ale/formatters/isis_formatter.py

```diff
@@ -0,0 +1,73 @@
+import json
+
+from ale.transformation import FrameNode
+from ale.rotation import ConstantRotation, TimeDependentRotation
+
+def to_isis(driver):
+    meta_data = {}
+
+    meta_data['CameraVersion'] = driver.sensor_model_version
+
+    meta_data['NaifKeywords'] = driver.isis_naif_keywords
+
+    j2000 = driver.frame_chain
```

Comment:
I was under the impression that this was being implemented in NaifSpice (https://github.com/USGS-Astrogeology/ale/issues/164 and https://github.com/USGS-Astrogeology/ale/pull/176)?

Reply:
Now that I am looking at an actual implementation, I don't think we need the sensor orientation or anything about rotations besides the frame chain in any of the data classes. The frame chain should has all of the time and rotation information baked into it already.